public/data/PritamSaha7.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Pritam Saha",
+    "type": "personal",
+    "bio": "I am pritam Saha, Computer Science graduate from Adamas University",
+    "avatar": "https://github.com/kaiwalyakoparkar.png",
```

Comment:
It seems this is the wrong link.
```suggestion
    "avatar": "https://github.com/Saha-7.png",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable